apps/public-docsite-v9/.storybook/main.js

```diff
@@ -40,4 +40,12 @@ module.exports = /** @type {Omit<import('../../../.storybook/main'), 'typescript
 
     return localConfig;
   },
+  refs: {
```

Comment:
asked async but reposting for better visibility: will this impact our sb docs load performance on local machine ? ty

Reply:
I haven't observed any changes in load performance, likely due to the fact that it's being lazy-loaded in the browser.

**Without refs:**
<img width="886" alt="Screenshot 2024-11-01 at 13 16 43" src="https://github.com/user-attachments/assets/22aa4fe9-3948-4a18-956d-24ef7aded8b6">


**With refs:**
<img width="886" alt="Screenshot 2024-11-01 at 13 15 54" src="https://github.com/user-attachments/assets/f558841c-9d3c-4dd4-b1ec-6e148ddf392a">

Reply:
awesome , ty for double check !